curriculum/challenges/english/14-responsive-web-design-22/learn-css-colors-by-building-a-set-of-colored-markers/61764c602bee6974e7790f35.md

```diff
@@ -13,6 +13,8 @@ To give the markers different colors, you will need to add a unique class to eac
 <div class="animal dog">
 ```
 
+On the HTML side, the order of multiple classes does not matter. But if the classes are styled the same way with CSS, the last class to be styled wins.
```

Comment:
"The last class to be styled wins" – I think this could be worded more clearly.

Perhaps you could say: "If you add multiple classes to an HTML element, the styles of the first classes you list may be overridden by later classes."

Reply:
I've now done this